ui/tract-renderer/src/main/java/org/cru/godtools/tract/widget/PageContentLayout.java

```diff
@@ -406,6 +431,69 @@ protected LayoutParams generateLayoutParams(final ViewGroup.LayoutParams p) {
         return generateDefaultLayoutParams();
     }
 
+    //region Card Bounce Animation
+
+    /**
+     * This method will either create a new bounce Animation or start the one that is already created.
+     */
+    @UiThread
+    private void animateFirstCardView() {
+        if (mBounceAnimator == null) {
+            for (int i = 0; i < getChildCount(); i++) {
+                final View child = getChildAt(i);
+                final LayoutParams lp = (LayoutParams) child.getLayoutParams();
+                switch (lp.childType) {
+                    case CHILD_TYPE_CARD:
+                        bounceCardAnimation(child);
+                        return; // Stop after first Card View
+                }
+            }
+        } else {
+            mBounceAnimator.start();
+        }
+    }
+
+    /**
+     *  This method will check to see if the Animate Card feature has been detected and start or stop the
+     *  animation if needed.
+     */
+    public void shouldAnimateCard() {
+        mShouldAnimateCard = !mSettings.isFeatureDiscovered(Settings.FEATURE_ANIMATE_CARD);
+        if (mShouldAnimateCard) {
+            bounceAnimationRunnable.run();
+        } else {
+            stopBounceAnimation();
+        }
+    }
+
+    /**
+     * This method will stop the bounce Animation and set the Feature as Discovered.
+     */
+    private void stopBounceAnimation() {
+
+        if (mBounceAnimator != null) {
+            mBounceAnimator.cancel();
+        }
+        mSettings.setFeatureDiscovered(Settings.FEATURE_ANIMATE_CARD);
+    }
+
+    /**
+     * This method will create a new instance of the bounce animation.
+     * @param targetView the Card View
+     */
+    @UiThread
+    private void bounceCardAnimation(final View targetView) {
+
+        mBounceAnimator = ObjectAnimator.ofFloat(targetView, "translationY",
+                targetView.getY(), targetView.getY() + 30, targetView.getY());
+        mBounceAnimator.setInterpolator(BounceUtil::getBounceInOut);
+        mBounceAnimator.setStartDelay(500);
+        mBounceAnimator.setDuration(1500);
```

Comment:
These durations should be constants at the top of the class so they are easy to change